docs/multiple-remotes.md

```diff
@@ -0,0 +1,34 @@
+## Pushing to multiple remotes
```

Comment:
I think it would be clearer to rename the title to "Using Gitkube with external git providers (like Github)"